resources/user-agents/browsers/firefox/firefox-0-x.json

```diff
@@ -1,6 +1,6 @@
 {
   "division": "Firefox #MAJORVER#.#MINORVER#",
-  "versions": ["0.10", "0.9", "0.8", "0.7", "0.6"],
```

Comment:
Since Firefox was Firebird for versions 0.7 and 0.6, I removed those versions from this file.